resources/windows/wix/project.wxs.erb

```diff
@@ -55,16 +55,22 @@
     <!-- INSTALLDIR -->
     <CustomAction Id="SaveCmdLineInstallDir" Property="CMDLINE_INSTALLDIR" Value="[INSTALLDIR]" Execute="firstSequence" />
     <CustomAction Id="SetFromCmdLineInstallDir" Property="INSTALLDIR" Value="[CMDLINE_INSTALLDIR]" Execute="firstSequence" />
+    <%= File.read("resources/windows/CustomActions") if File.exist?("resources/windows/CustomActions") -%>
```

Comment:
The output looks like https://gist.github.com/melissa/23c6ca022c3aa7375c64.

If you don't like this, by all means give me a better approach